docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>This seems rather complicated. I'm wondering how necessary it is to make these functions considering that we're only using it once?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>